app/controllers/SR20.scala

```diff
@@ -44,12 +58,16 @@ class SR20 @Inject() (val errorHandler: ErrorHandler,
                       val cookieSigner: CookieSigner,
                       confirmApisView: ConfirmApisView,
                       turnOffApisMasterView:TurnOffApisMasterView,
-                      val apmConnector: ApmConnector)
+                      val apmConnector: ApmConnector,
+                      responsibleIndividualView: ResponsibleIndividualView,
+                      flowService: GetProductionCredentialsFlowService)
                      (implicit val ec: ExecutionContext, val appConfig: ApplicationConfig)
   extends ApplicationController(mcc)
      with CanUseCheckActions{
-       
-def confirmApiSubscriptionsPage(sandboxAppId: ApplicationId): Action[AnyContent] = whenTeamMemberOnApp(sandboxAppId) { implicit request =>
```

Comment:
I think these need to limit to admin only.  Can't remember the actionBuilder name right now.